server/config/config.go

```diff
@@ -697,3 +705,61 @@ func TestConfig() FleetConfig {
 		},
 	}
 }
+
+func TestKinesisPluginConfig() FleetConfig {
```

Comment:
Could we move this to `app_test.go`? If it's used from multiple places, I prefer to have these in a `testing_utils.go` file, but otherwise, as close to the test as possible.